pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/OffloadPoliciesImpl.java

```diff
@@ -257,6 +261,14 @@ public static OffloadPoliciesImpl create(Properties properties) {
                 }
             }
         });
+        Map<String, String> extraConfigurations = properties.entrySet().stream()
+            .filter(entry -> entry.getKey().toString().startsWith("managedLedgerOffloadExtraConfig"))
```

Comment:
Could we add a final static string for this prefix? We may need to add a description in the doc.

Reply:
Since the prefix is not exposed for others to use and there have others also using string https://github.com/apache/pulsar/blob/4c69584a888f1365cd1bd78e0dbd74d10a0aa413/pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/OffloadPoliciesImpl.java#L337. I can improve them all in another PR. 

The document will be auto-generated with the admin cli, like other policies: https://pulsar.apache.org/reference/#/3.0.x/pulsar-admin/namespaces?id=set-offload-policies.
We don't need to add it to other places. Do you know if any places need to add this as well?

Reply:
Got it, we can add it to cmd in the future.